sqlite/migrations/all.go

```diff
@@ -0,0 +1,13 @@
+//go:generate env GO111MODULE=on go run github.com/kevinburke/go-bindata/go-bindata -prefix "migrations/" -o ./migrations_gen.go -ignore README|go|conf -pkg migrations .
```

Comment:
Not doing anything too fancy with this since I hope to replace it with `go:embed`.